spec/api-menu-spec.js

```diff
@@ -284,6 +284,17 @@ describe('Menu module', () => {
       return closeWindow(w).then(() => { w = null })
     })
 
+    it('should emit menu-will-show event', (done) => {
+      menu.on('menu-will-show', () => { done() })
```

Comment:
The docs say the it should work on Mac only, but it seems like it works everywhere. How is that? )

Reply:
oh oops i copied that incorrectly

Reply:
@alexeykuzmin it's fixed now